tables/AlternateMans/test/tColumnarFile.cc

```diff
@@ -0,0 +1,164 @@
+#include <boost/test/unit_test.hpp>
+
+#include "../StokesIStMan/SimpleColumnarFile.h"
+#include "../StokesIStMan/BufferedColumnarFile.h"
+
+using casacore::VarBufferedColumnarFile;
+using casacore::SimpleColumnarFile;
+
+template<typename ColumnarFile>
+void TestEmptyConstructor() {
+  ColumnarFile file;
+  BOOST_CHECK_EQUAL(file.NRows(), 0);
+  BOOST_CHECK_EQUAL(file.Filename(), "");
+  BOOST_CHECK(!file.IsOpen());
+
+  file = ColumnarFile();
+  BOOST_CHECK_EQUAL(file.NRows(), 0);
+  BOOST_CHECK_EQUAL(file.Filename(), "");
+  BOOST_CHECK(!file.IsOpen());
```

Comment:
This second part of the test checks that assigning an empty ColumnarFile to an existing empty ColumnarFile yields an empty ColumnarFile. I don't see much value in this part. If you want to test assignment, please use two different objects in the assignment.

Reply:
Fair, done.